examples/coronavirustwittermap/web/app/websocket/TwitterMapServerToCloudBerrySocket.scala

```diff
@@ -68,24 +84,24 @@ class TwitterMapServerToCloudBerrySocket(out: ActorRef, config: Configuration) {
                 /**
                   * 1) pinMapResult response from Cloudberry has format as follows,
                   * { id: "defaultID",
-                  *   category: "pinMapResult",
-                  *   value: [[
-                  *     { id: 1236910669389131776, place.bounding_box: [[-120.582586,46.958017], [-120.49726,47.028542]] },
-                  *     { id: 1236509727560880128, coordinate: [-121.944442,37.325656], place.bounding_box: [[...]]},
-                  *     ... ...
-                  *   ]]
+                  * category: "pinMapResult",
```

Comment:
Let's avoid this unnecessary format changes, and also this indent is useful for readers.

Reply:
Sorry, I didn’t change this part must have been due to reformat code option